bot/src/commands/add.js

```diff
@@ -0,0 +1,40 @@
+const { getFeed } = require("../utilities/getFeed");
+
+module.exports = async (args, guildId, client) => {
+  const [url] = args;
+  if (!url) {
+    return `Usage: @RnD add <rss URL>
+      Ex: @RnD add <https://xkcd.com/rss.xml>`;
```

Comment:
This might be a good place to consider throwing a custom error. Something like `ArgumentError` with this string as the `message` property.